srcf/__init__.py

```diff
@@ -2,20 +2,25 @@
 
 """SRCF python library for common actions in maintenance scripts"""
 
+import warnings
+
 # Python 2 & 3 compatability
 import six as __six
```

Comment:
why do we even rename this import?

Reply:
Dunder-mangling is only a class thing, and we already maintain `__all__` ourselves, so there's no real need for this.